lib/elixir/lib/map.ex

```diff
@@ -885,8 +887,9 @@ defmodule Map do
       {1, %{}}
 
   """
-  @spec get_and_update!(map, key, (value -> {get, value} | :pop)) :: {get, map}
-        when get: term
+  @spec get_and_update!(map, key, (value -> {current_value, new_value :: value} | :pop)) ::
+          {current_value, map}
+        when current_value: value
```

Comment:
Can we keep those two lines together? Or will the formatter complain?

Reply:
The formatter splits them